package.json

```diff
@@ -3,10 +3,12 @@
   "version": "2.2.1",
   "description": "",
   "main": "lib/canopy-sofe-extensions.js",
+  "browserslist": [
+    "extends browserslist-config-canopy"
+  ],
   "scripts": {
-    "build": "webpack --config webpack.config.js",
-    "watch": "webpack --config webpack.config.js -w",
-    "prepublish": "npm run build",
+    "build": "babel src --out-dir lib --source-maps",
```

Comment:
i don't think there's anything we need webpack for here -- we can just babel the file and let `common-dependencies` webpack the library